src/sentry/api/endpoints/organization_missing_org_members.py

```diff
@@ -161,28 +172,9 @@ def provider_reducer(dict, integration):
                 continue
 
             queryset = _get_missing_organization_members(
-                organization, integration_provider, integration_ids
+                organization, integration_provider, integration_ids, shared_domain
             )
 
-            if shared_domain:
-                queryset = queryset.filter(email__endswith=shared_domain)
-            else:
-                for filtered_email in FILTERED_EMAIL_DOMAINS:
-                    queryset = queryset.exclude(email__endswith=filtered_email)
-
-            for filtered_character in FILTERED_CHARACTERS:
-                queryset = queryset.exclude(email__icontains=filtered_character)
-
-            if queryset.exists():
-                query = request.GET.get("query")
-                if query:
-                    tokens = tokenize_query(query)
-                    if "query" in tokens:
-                        query_value = " ".join(tokens["query"])
-                        queryset = queryset.filter(
-                            Q(email__icontains=query_value) | Q(external_id__icontains=query_value)
```

Comment:
Are there no requests that currently have this query parameter? Also, doesn't removing this make it so we don't filter bots out and such

Reply:
no requests. the query is only for if somebody wants to filter for commit authors with a specific `email` or `external_id`. we automatically filter out a few email domains and characters we've deemed as invalid

Reply:
we don't have a search bar on the modal so this wouldn't be used